engine/src/main/java/org/hibernate/validator/internal/metadata/core/BuiltinConstraint.java

```diff
@@ -64,6 +64,7 @@
 	ORG_HIBERNATE_VALIDATOR_CONSTRAINTS_MOD10_CHECK("org.hibernate.validator.constraints.Mod10Check"),
 	ORG_HIBERNATE_VALIDATOR_CONSTRAINTS_MOD11_CHECK("org.hibernate.validator.constraints.Mod11Check"),
 	ORG_HIBERNATE_VALIDATOR_CONSTRAINTS_MOD_CHECK("org.hibernate.validator.constraints.ModCheck"),
+	ORG_HIBERNATE_VALIDATOR_CONSTRAINTS_NORMALIZED("org.hibernate.validator.constraints.Normalized"),
```

Comment:
Could you check out the comment at the top of this class? You need to add something in 2 tests validating things for all the constraints. The first one is not critical for your constraint as you don't have a parameter in your message but better be safe than sorry if things change later.

Thanks!

Reply:
Added :+1: